transip/client.py

```diff
@@ -46,14 +47,18 @@ def __init__(self, service_name, login, private_key_file='decrypted_key', endpoi
 
     def _sign(self, message):
         """ Uses the decrypted private key to sign the message. """
-        with open(self.private_key_file) as private_key:
-            keydata = private_key.read()
-            privkey = rsa.PrivateKey.load_pkcs1(keydata)
-            signature = rsa.sign(message.encode('utf-8'), privkey, 'SHA-512')
-            signature = base64.b64encode(signature)
-            signature = quote_plus(signature)
-
-        return signature
+        if os.path.exists(self.private_key_file):
+            with open(self.private_key_file) as private_key:
+                keydata = private_key.read()
+                privkey = rsa.PrivateKey.load_pkcs1(keydata)
+                signature = rsa.sign(message.encode('utf-8'), privkey, 'SHA-512')
+                signature = base64.b64encode(signature)
+                signature = quote_plus(signature)
+
+            return signature
+        else:
+            print("The private key does not exist.")
```

Comment:
Can you add `from __future__ import print_function` at the top of this file so that we can support Python 2.7 as well? This will fix the failing test. Thanks.